### DISCUSSION TITLE
Demos error

### DISCUSSION THREAD

**Discussion by Nannancao** [CERN/TIGRE#446] [Q&A]
hi,I configured the environment and code as instructed, but I did not achieve the desired results when I ran the demos, for example:In d01:there is a AttributeError: module 'numpy' has no attribute 'float',and my numpy = 1.24.1,In d02:there is only the result of (64, 64, 64) and no figures, images.I looked up the definition of the function including PlotImg, Plotpro and Plot*，Will some images come out when I use these functions?e.g. In d02：tigre.plotImg(shepp, dim="z")，tigre.plotImg(head, dim="z") and some examples in other demos. I was saddened to find that there were no result images when I ran the code.Can you help me answer my doubts? Thank you so much

**Comment by AnderBiguri** [comment]
Hi @Nannancao , numpy 1.24 recently released and we have had some problems with the update. An older version of `numpy` should definetly work well, but I will check the demos to try to fix it ASAP. 

However demo 2 should definetly plot images using `numpy`. Are you running your code in a machine with a graphics?

**Comment by Nannancao** [comment]
I checked and there was indeed a missing package for graphics，i will install the relavant package and may need your help later. Thank you very much. This has been very helpful.
